lib/ext.core.TemplateHandler.js

```diff
@@ -107,7 +107,14 @@ TemplateHandler.prototype.onTemplate = function ( token, frame, cb ) {
 	// in <infobox> extension
 	frame.templateArgInfo = frame.templateArgInfo || {};
 	if(tgt && tgt.target) {
-		frame.templateArgInfo[tgt.target] = this.getArgInfo(state);
+		// infobox template can be used multiple times in single article
```

Comment:
@inez: I'm not happy with this hot fix. With your knowledge about Parsoid, do you have any suggestions  how we could refactor this part?